README.md

```diff
@@ -45,27 +35,4 @@ Run `pipenv install -d` to install all packages needed
   * Common dependencies pre-configured, split into regular and development packages
   * Testing, linting, code coverage, and uwsgi configuration
 
-## Configuration
-
-This section will lay out all the changes that need to be performed to the template configuration to use with a new app.
-
-| File                    | Line | Description                                                                         |
-|-------------------------|------|-------------------------------------------------------------------------------------|
-| .circleci/config.yml    | 5    | Change pennlabs/example-project to the name of the new app to publish to docker hub |
-| .circleci/config.yml    | 45   | In pennlabs.settings.ci, change pennlabs to the name of your django project         |
-| .circleci/config.yml    | 119  | Change example-deploy to the CircleCI context containing production secrets         |
-| settings/base.py        | 55   | In pennlabs.urls, change pennlabs to the name of your django project                |
-| settings/base.py        | 73   | In pennlabs.wsgi.application, change pennlabs to the name of your django project    |
-| settings/base.py        | 127  | Change example.pennlabs.org to the FQDN used in production                          |
-| settings/base.py        | 128  | Change example_admin to the admin tag desired from platform                         |
-| settings/ci.py          | 1    | In pennlabs.settings.base, change pennlabs to the name of your django project       |
-| settings/development.py | 3    | In pennlabs.settings.base, change pennlabs to the name of your django project       |
-| settings/production.py  | 6    | In pennlabs.settings.base, change pennlabs to the name of your django project       |
-| settings/staging.py     | 4    | In pennlabs.settings.base, change pennlabs to the name of your django project       |
-| settings/production.py  | 18   | Change ALLOWED_HOSTS to reflect the FQDN used in production                         |
-| settings.cfg            | 9    | Change the modules on this line to the ones created in your django project          |
-| settings.cfg            | 21   | In pennlabs.wsgi:application, change pennlabs to the name of your django project    |
-| urls.py                 | 5    | Change Pennlabs Example Admin to a descriptive name of your django project          |
-| k8s/values.yaml         | 5    | Change Pennlabs Example Admin to a descriptive name of your django project          |
```

Comment:
I think we should still keep some of the k8s instructions because we aren't auto setting all of those.

Reply:
deferring again to @pawalt here

Reply:
I think k8s docs don't belong here. These settings are going to be pretty similar across products, so i'd rather make a general guide in Notion.

Reply:
I think it may make more sense to create a script to do all the replacing for a new project and just include that instead.